docs/components/arm/_index.md

```diff
@@ -19,54 +19,20 @@ outputs:
+Arms provide an API for linear motion planning, including self-collision prevention and obstacle avoidance.
 
-In simple terms, an _arm_ has two ends: one fixed in place, and one with a device you can position.
+If you have a physical robotic arm, consisting of a serial chain of joints and links, with a fixed end and an end effector end, use an arm component.
 
-When controlling an arm component, you can place the end effector at arbitrary cartesian positions relative to the base of the arm.
-You can do this by calling the `MoveToPosition` method to move the end effector to specified cartesian coordinates, or by controlling the joint positions directly with the `MoveToJointPositions` method.
-
-When controlling an arm with `viam-server`, the following features are implemented for you:
-
-- Linear motion planning
-- Self-collision prevention
-- Obstacle avoidance
-
-## Motion planning with your arm's built-in software
-
-Each arm model is supported with a driver that is compatible with the software API that the model's manufacturer supports.
-While some arm models build inverse kinematics into their software, many do not.
-
-- Most of the arm drivers for the Viam RDK bypass any onboard inverse kinematics, and use Viam's [motion service](/services/motion/) instead.
-
-- This driver handles turning the arm on and off, querying the arm for its current joint position, sending requests for the arm to move to a specified set of joint positions, and engaging brakes as needed, if supported.
-
-Arm drivers are also paired, in the RDK, with JSON files that describe the kinematics parameters of each arm.
-
-- When you configure a supported arm model to connect to `viam-server`, the Arm driver will load and parse the kinematics file for the Viam RDK's [frame system](/services/frame-system/) service to use.
-
-- The [frame system](/services/frame-system/) will allow you to easily calculate where any part of your machine is relative to any other part, other machine, or piece of the environment.
-
-- All arms have a `Home` position, which corresponds to setting all joint angles to 0.
-
-- When an arm is moved with a `move_to_position` call, the movement will follow a straight line, and not deviate from the start or end orientations more than the start and orientations differ from one another
-
-- If there is no way for the arm to move to the desired location in a straight line, or if it would self-collide or collide with an obstacle that was passed in as something to avoid, then the `move_to_position` call will fail.
-
-## Related services
-
-{{< cards >}}
-{{< relatedcard link="/services/motion/" >}}
-{{< relatedcard link="/services/frame-system/" >}}
-{{< relatedcard link="/services/data/" >}}
-{{< /cards >}}
+Typically, arms have two ends: one fixed in place, and one with a device you can position.
+When operating an arm you can place its end effector at arbitrary cartesian positions relative to the base of the arm.
```

Comment:
```suggestion
When controlling an arm, you can place its end effector at arbitrary cartesian positions relative to the base of the arm.
```
"controlling" seems to me like a more correct word here because it gets more to the point of the API, whereas in my head "operating" somehow implies driving it with a joystick or similar.